client/app/states/designer/blueprints/editor/editor.state.js

```diff
@@ -69,16 +69,26 @@
     if (blueprint) {
       if (blueprintTags && blueprintTags.resources) {
         blueprint.tags = blueprintTags.resources;
+        angular.forEach(blueprint.tags, processTag);   // <--- Add this line
       }
```

Comment:
Sorry to be a pain in the arse, but you do not need the "// <--- Add this line" comment :-)